fakepkg

```diff
@@ -142,11 +141,13 @@ for ARG in "$@"; do
 done
 
 # Assembling PKG_LIST and DEST_DIR
-PARAMS=$(getopt -o o:j:vV --long out:,jobs:,verbose,version -n "$myname" -- "$@")
+PARAMS=$(getopt -o o:j:r:v --long out:,jobs:,root:,verbose -n "$myname" -- "$@")
```

Comment:
Is there a reason for leaving out the `Version` option or did this just happen by accident?

Reply:
Sorry, yes, my bad. Actually I was doing the original changes to the older version as you did not bump the version since the last modifications. Then realizing the issue I was redoing the changes against the master. And clearly missed this one..

Reply:
Ah, I see. No problem.